torch/autograd/__init__.py

```diff
@@ -213,7 +213,8 @@ def grad(
     create_graph: bool = False,
     only_inputs: bool = True,
     allow_unused: bool = False,
-    is_grads_batched: bool = False
+    is_grads_batched: bool = False,
+    zero_grad_unused: bool = False,
```

Comment:
A few other options for the name:
- materialized_unused
- materialized_grad

@albanD any opinions on this one?

Reply:
I prefer materialize_grad to match the custom Function argument naming.

Reply:
The new parameter was renamed to `materialize_grad` in https://github.com/pytorch/pytorch/pull/97015/commits/36b141ad1709c895899346e33ad7e6d54d128989